transports/sql/discard-old-messages_receiving_sqltransport_[2,3).partial.md

```diff
@@ -0,0 +1,3 @@
+The messages with specified Time-To-Be-Received (TTBR) are pushed through the message pump, just like regular messages, irrespectively if their TTBR already elapsed or not. The receive query returns the difference between the expiry time and the current UTC time in the database so that the transport can make a decision if a given message has expired.
```

Comment:
> so that the transport can make a decision if a given message has expired.

It feels it's missing an extra sentence explaining what happens when the transport makes that decision? Is that the point where it can decide to NOT send it? (I'm thinking about something similart to "The messages with elapsed Time-To-Be-Received (TTBR) are ignored by the message pump whose receive query has a `where` clause that excludes expired messages.")